addons/sourcemod/translations/ttt.phrases.txt

```diff
@@ -239,7 +239,7 @@
 
 	"The weapon used has been: himself (suicide)"
```

Comment:
Could you correct the phrase as well here.

Reply:
I may be mistaken, but from what I gathered, it seemed like the strings prefacing bracket blocks are the internal phrases used to identify the actual block of localizable strings that gets shown in game. If this assumption is correct, then changing this block (1) would require additional refactoring inside of the actual code and (2) would not make any visible difference to a player. If I'm misinterpreting the way localizable strings are handled, please let me know so I can correct these remaining blocks.

Reply:
It isnt shown to the user, just thought it will be best to rectify it in the code as well.

Reply:
I agree, though those changes would be a bit more substantial and I'd want to make sure I don't miss anything. I'll take a look at it later if I get a chance.